x/tokenomics/types/expected_keepers.go

```diff
@@ -56,7 +56,7 @@ type ProofKeeper interface {
 	GetProof(ctx context.Context, sessionId, supplierOperatorAddr string) (proof prooftypes.Proof, isProofFound bool)
 	RemoveProof(ctx context.Context, sessionId, supplierOperatorAddr string)
 
-	AllClaims(ctx context.Context, req *prooftypes.QueryAllClaimsRequest) (*prooftypes.QueryAllClaimsResponse, error)
+	GetSessionEndHeightClaimsIterator(ctx context.Context, sessionEndHeight int64) *prooftypes.ClaimsIterator
```

Comment:
Do we need to run mockgen to generate those artifacts?

Reply:
Even if `AllClaims` is never used in in `tokenomics` module tests and thus `GetSessionEndHeightClaimsIterator` too.

Mocks are indeed generated for the expected `ProofKeeper`.